eve/workers/end2end.yaml

```diff
@@ -4,6 +4,18 @@ kind: Pod
 metadata:
   name: "zenko-operator-worker-smoke"
 spec:
+  initContainers:
```

Comment:
Do you think you could add some context to this, it looks pretty magical. Or maybe a follow up ticket to remove it when we can.

Reply:
Just added the link to the incident